src/features/client/pages/CourierRegisterLanding/index.jsx

```diff
@@ -0,0 +1,312 @@
+            <Card textAlign="center" alignItems="center" width="300px">
+              <CardBody>
+                <Text>
+                  &quot;As a student, free schedule gives me the opportunity to
+                  earn extra income while maintaining the highest level of
+                  education.&quot;
+                </Text>
+              </CardBody>
+              <CardFooter>
+                <Box>
+                  <Text>Name</Text>
+                  <Text>City</Text>
+                </Box>
+              </CardFooter>
+            </Card>
+            <Card textAlign="center" alignItems="center" width="300px">
+              <CardBody>
+                <Text>
+                  &quot;Freedom of choice is one of my core values. I get the
+                  opportunity to be my own boss, to work whenever I want and to
+                  manage my income easily. I am really happy with this
+                  platform.&quot;
+                </Text>
+              </CardBody>
+              <CardFooter>
+                <Box>
+                  <Text>Name</Text>
+                  <Text>City</Text>
+                </Box>
+              </CardFooter>
+            </Card>
+            <Card textAlign="center" alignItems="center" width="300px">
+              <CardBody>
+                <Text>
+                  &quot;Wolt gives me the opportunity to earn extra income in my
+                  spare time. It&apos;s also fun to drive around the city, take
+                  in the sights and meet people. I have been a Wolt Courier
+                  Partner for a year now and I can&apos;t complain about
+                  anything.&quot;
+                </Text>
+              </CardBody>
+              <CardFooter>
+                <Box>
+                  <Text>Name</Text>
+                  <Text>City</Text>
+                </Box>
+              </CardFooter>
+            </Card>
+          </Flex>
+        </Box>
+        <Box m={10}>
+          <Flex flexDirection="column" alignItems="center">
+            <Text as="b" fontSize="4xl">
+              Sign up today and you&apos;ll hit the road in no time!
+            </Text>
+            <Button width="150px" m={5}>
+              <Link to="/apply-as-courier">Submit A Form</Link>
+            </Button>
+          </Flex>
+        </Box>
+        <Box m={10}>
+          <Box textAlign="center">
+            <Text as="b" fontSize="4xl">
+              Frequently asked questions
+            </Text>
+          </Box>
+          <Box mb={5}>
+            <ExtendedComponent
+              title={'How long does the application process take?'}
+            >
+              Typically, it takes less than 30 minutes to fill out the
+              application. If you&apos;re new to Wolt, you&apos;ll need to
+              create an account, enter in a few basic details, and upload the
+              required documents.
+            </ExtendedComponent>
```

Comment:
1. This should be called accordion item, not extended component
2. https://chakra-ui.com/docs/components/accordion chakra has it own implementation of accordions. Refactor please